utils/paramfetch/src/lib.rs

```diff
@@ -98,40 +86,19 @@ pub async fn get_params(
             SectorSizeOpt::All => true,
         })
         .for_each(|(name, info)| {
-            let cmb = mb.clone();
             let data_dir_clone = data_dir.to_owned();
             tasks.push(task::spawn(async move {
-                if let Err(e) =
-                    fetch_verify_params(&data_dir_clone, &name, Arc::new(info), cmb).await
-                {
+                if let Err(e) = fetch_verify_params(&data_dir_clone, &name, Arc::new(info)).await {
                     warn!("Error in validating params {}", e);
                 }
             }))
         });
 
-    if let Some(multi_bar) = mb {
-        let cmb = multi_bar.clone();
-        let (mb_send, mb_rx) = bounded(1);
-        let mb = task::spawn(async move {
-            while mb_rx.try_recv().is_err() {
-                cmb.listen();
-                task::sleep(Duration::from_millis(1000)).await;
-            }
-        });
-        for t in tasks {
-            t.await;
-        }
-        mb_send
-            .send(())
-            .await
-            .expect("Receiver should not be dropped");
-        mb.await;
-    } else {
-        for t in tasks {
-            t.await;
-        }
+    for t in tasks {
+        t.await;
     }
 
+    // XXX: We return OK(()) even if 'fetch_verify_params' fails. :-/
```

Comment:
Can we do something about it? Or at least create an issue?

Reply:
Tracking issue created.